### ISSUE TITLE
U+1BAB SUNDANESE SIGN VIRAMA does not form conjuncts

### ISSUE THREAD

**Issue by dscorbett** [googlei18n/noto-fonts#1100]
### Font
NotoSansSundanese-Regular.ttf

### Where the font came from, and when
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansSundanese-unhinted.zip
Date: 2018-03-19

### Font version
Version 1.05 uh

### Issue
U+1BAB SUNDANESE SIGN VIRAMA should form conjuncts. Currently, it appears as a small plus sign overstriking the following glyph. See [L2/09-251R](http://www.unicode.org/L2/L2009/09251r-n3666r-sundanese.pdf) for examples of conjuncts.

### Character data
ᮊ᮫ᮒ
U+1B8A SUNDANESE LETTER KA
U+1BAB SUNDANESE SIGN VIRAMA
U+1B92 SUNDANESE LETTER TA

### Screenshot
![x](https://user-images.githubusercontent.com/1124347/37609901-ab2eec64-2b74-11e8-9ebb-b3946ba2d6b8.png)

**Comment by Ilhamkang** [comment]
I agree to add conjuncts based L2/09-251R document. I will be glad to check all sequences.
